tests/cli/test_cli.py

```diff
@@ -53,6 +53,7 @@ def test_cli_clip_help(tmpdir):
     )
 
 
+@pytest.mark.skip(reason="Needs implementation of RasterDataSet.")
```

Comment:
I don't see this skip for MeshModel, is xugrid properly imported in the test environment?

Reply:
Sorry I don't understand the question. This test is skipped because we haven't implemented Rasterdataset yet. What does that have to do with the MeshModel?